aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md

```diff
@@ -227,15 +227,15 @@ Add the following `Page_Init` method, which enables Dynamic Data functionality.
 
 Run the *OfficeAssignments.aspx* page.
 
-[![Image10](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image32.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image31.png)
+[![A screenshot that shows the OfficeAssignments page.](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image32.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image31.png)
```

Comment:
Let's put a space between Office and Assignments so the screen reader will read it correctly.